src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -185,12 +185,6 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges, O
     });
     this.filterHiddenColumns();
     // Load the data table content every N ms or at least once.
-    // Force showing the loading indicator if there are subscribers to the fetchData
-    // event. This is necessary because it has been set to False in useData() when
-    // this method was triggered by ngOnChanges().
-    if (this.fetchData.observers.length > 0) {
```

Comment:
You have to re-add this code part, otherwise the loading indicator won't be displayed anymore, check Pool/OSD/RGW user/... list views.

Reply:
I think we should add additional unit tests for that issue to make sure it won't be broken by any future changes.

Reply:
I've reverted this change, and since you already have an open PR to fix this progress bar issue, I've marked this PR to be merged after https://github.com/ceph/ceph/pull/24016